### PR TITLE
Adding clear button

### PR DESCRIPTION
**Pull request by jusabatier**
Add a "Clear" button to clear active fields and perform a new search

### CODE REVIEW

File: mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js [hunk 1]
```diff
@@ -157,13 +157,24 @@ GEOR.Addons.Cadastre = Ext.extend(GEOR.Addons.Base, {
...
-                    handler: function() {
-                        this.win.hide();
-                    },
-                    scope: this
-                }],
+                fbar: [
+                    this.cbx, 
+                    '->', 
+                    {
+						text: OpenLayers.i18n("Clean"),
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js]: indents are not ok

File: mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js [hunk 2]
```diff
@@ -157,13 +157,24 @@ GEOR.Addons.Cadastre = Ext.extend(GEOR.Addons.Base, {
...
+						handler: function() {
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js]: handler: this.cleanActiveFields

File: mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js [hunk 3]
```diff
@@ -178,6 +189,18 @@ GEOR.Addons.Cadastre = Ext.extend(GEOR.Addons.Base, {
         this.map.addLayer(this.layer);
         this.win.show();
     },
+    
+    cleanActiveFields: function() {
+		var fields = this.win.items.items[0].activeTab.form.items.items;
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js]: there should be a better way to get these fields ...

**Comment by fvanderbiest** [comment]
Thanks for the interesting contribution :-)
Some minor comments to be addressed before merge.

**Comment by jusabatier** [comment]
Thanks for suggestion, I've troubles with indentation when copy/paste from notepad x)

Effectively there were a better way to get fields, easiest to find it with clear head ;)

Now it should be OK

**Comment by fvanderbiest** [comment]
Looks good, thanks

**Pull request merged**